packages/react/src/themes/teams/getPointerWithSvgStyles.ts

```diff
@@ -0,0 +1,55 @@
+import { PopperChildrenProps } from '../../lib/positioner'
+
+const rtlMapping = {
+  left: 'right',
+  right: 'left',
+}
+
+const getPointerWithSvgStyles = (
```

Comment:
Should be refactored, there are lots of duplications with the `getPointerStyles.ts`